### ISSUE TITLE
Studies: ARIA input field does not have accessible name

### ISSUE THREAD

**Issue by RyanAJensen** [pulibrary/DSS#572]
### Steps to recreate
Visit /studies/new or /studies/edit and run the Axe dev tools

### URL
http://localhost:3000/studies/new
http://localhost:3000/studies/edit

### Issues
"Select the Resource this Study is associated with" input field does not have an accessible name

### Severity
High

### WCAG Success Criterion
- [ ] ARIA input fields must have an accessible name (2)

### WCAG Conformance level (A, AA, AAA)
A

### User impact
When ARIA input fields do not have an label, screen reader users do not know the input data expectations.

### Recommended fix
To solve this problem, you need to fix at least (1) of the following:
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] WCAG Success Criterion are resolved

### Notes
The two instances seem to be the same select box
